src/core/valuemapmodel.h

```diff
@@ -50,26 +46,20 @@ class ValueMapModel : public QAbstractListModel
       ValueRole                   //!< obtain the value
     };
 
-    /**
-     * Create a new value map model
-     */
+    Q_ENUM( ValueMapRoles )
+
     explicit ValueMapModel( QObject *parent = nullptr );
 
     /**
      * The map, see the property description
```

Comment:
Can you update these to rely on the proper `//! \copydoc ValueMapModel::valueMap`  for this function and the setMap function? We might as well take the opportunity to clean this up :)